doc/manual/metaprogramming.rst

```diff
@@ -415,11 +418,16 @@ Basics
 
 Here is an extraordinarily simple macro:
 
-.. doctest::
+.. testcode::
 
-    julia> macro sayhello()
-               return :( println("Hello, world!") )
-           end
+    macro sayhello()
+        return :( println("Hello, world!") )
+    end
+
+.. testoutput::
```

Comment:
Correct. The original doctest was failing since creating the macro now was outputting:
```
@sayhello (macro with 1 method)
```
Using `testcode` and `testoutput` allowed me to keep the example tested while keeping the documentation the same. Note we did lose the `julia>` prompt.